zipkin-lens/src/components/App/LanguageSelector.tsx

```diff
@@ -72,9 +72,9 @@ const LanguageSelector = () => {
         return;
       }
       setLocale(locale);
-      window.location.reload();
```

Comment:
Now we can change languages without `reload()`.

https://github.com/openzipkin/zipkin/assets/19551419/c173e834-2f72-47f8-a900-000bbd7903a7